src/github.com/getlantern/lantern/feed.go

```diff
@@ -157,6 +162,7 @@ func GetFeed(locale string, allStr string, proxyAddr string,
 		return
 	}
 
+	ioutil.WriteFile("/tmp/"+locale, contents, 0644)
```

Comment:
Oh, good catch!  That was just temporary to save off the test files, I'll remove.